sources/assets/zsh/history.d/nmap-parse-output

```diff
@@ -0,0 +1,36 @@
+# nmap-parse-output | grep -E '^  [a-z]' | sed -E -e 's/\[([^ ]+)\]/$\1/g' -e 's/\$service-name/$service_name/' | xargs -i sh -c "echo 'nmap-parse-output \$nmap_xml {}'"
```

Comment:
What is this comment for?

Reply:
It's there to show how the history is generated from the tool help. I figured it's safe to include because `add-history` strips comments.

Reply:
`add-history` will reference the file : `history.d/nmap-parse-output`
The contents of the file will be passed to history without strip

Reply:
you can remove the line then @compass-dexter

Reply:
Oh yeah, I misread that :)